### PR TITLE
Move installed root ssh authorized keys to a specific aytest file (bsc#1066342)

### PR DESCRIPTION
**Pull request by teclator**
See https://github.com/yast/yast-users/pull/150 & https://github.com/yast/aytests-tests/pull/105

### CODE REVIEW

File: package/rubygem-aytests.changes [hunk 1]
```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Tue Nov 28 21:15:24 UTC 2017 - knut.anderssen@suse.com
+
+- Use .ssh/authorized_keys2 instead of .ssh/authorized_keys in 
+  postinshtall.sh script (bsc#1066342)
```
Comment (imobachgs) [inline package/rubygem-aytests.changes]: typo: postinstall.sh

File: share/veewee/postinstall.sh [hunk 2]
```diff
@@ -10,6 +10,13 @@ echo 'solver.onlyRequires = true' >> /etc/zypp/zypp.conf
 # remove zypper package locks
 rm -f /etc/zypp/locks
 
+# Moved the root ssh authorized keys file if exist for being tested
+if [[ ! -f /root/.ssh/authorized_keys.aytests ]]; then
```
Comment (imobachgs) [inline share/veewee/postinstall.sh]: Why not combine both conditions?

File: package/rubygem-aytests.changes (hunk 1 above)
Comment (teclator) [reply package/rubygem-aytests.changes]: Updated changelog

File: share/veewee/postinstall.sh (hunk 2 above)
Comment (teclator) [reply share/veewee/postinstall.sh]: Why not :), updated

**Pull request merged**